cns/cnsclient/cnsclient.go

```diff
@@ -211,7 +211,7 @@ func (cnsClient *CNSClient) DeleteHostNCApipaEndpoint(networkContainerID string)
 }
 
 // RequestIPAddress calls the requestIPAddress in CNS
-func (cnsClient *CNSClient) RequestIPAddress(orchestratorContext []byte) (*cns.IPConfigResponse, error) {
+func (cnsClient *CNSClient) RequestIPAddress(ipconfig *cns.IPConfigRequest) (*cns.IPConfigResponse, error) {
```

Comment:
>RequestIPAddress [](http://example.com/codeflow?start=28&length=16)

Is this used for a debug API?

Reply:
This is used in the CNI call to CNS for ipam